packages/launchdarkly-adapter/index.ts

```diff
@@ -0,0 +1,5 @@
+// This file exists  because we want jest to use our non-compiled code to run tests
+// if this file is missing, and you have a `module` or `main` that points to a non-existing file
+// (ie, a bundle that hasn't been built yet) then jest will fail if the bundle is not yet built.
+// all apps should export all their named exports from their root index.js
+export * from './modules';
```

Comment:
As we do in appkit, this "proxy" file is necessary to allow typescript to find the exports of a package, if no `dist` is available.